app/scripts/modules/core/src/pipeline/config/triggers/pubsub/pubsub.trigger.ts

```diff
@@ -0,0 +1,44 @@
+import {IController, module} from 'angular';
+
+import {SETTINGS} from 'core/config/settings';
+import {PIPELINE_CONFIG_PROVIDER} from 'core/pipeline/config/pipelineConfigProvider';
+import {SERVICE_ACCOUNT_SERVICE, ServiceAccountService} from 'core/serviceAccount/serviceAccount.service';
+import {IPubsubTrigger} from '@spinnaker/core';
+
+class PubsubTriggerController implements IController {
+
+  public fiatEnabled: boolean = SETTINGS.feature.fiatEnabled;
+  public pubsubTypes = ['kafka', 'google'];
+  public serviceAccounts: string[] = [];
+
+  constructor(public trigger: IPubsubTrigger, private serviceAccountService: ServiceAccountService) {
+    'ngInject';
+
+    this.serviceAccountService.getServiceAccounts().then(accounts => {
+      this.serviceAccounts = accounts || [];
+    });
+  }
+}
+
+export const PUBSUB_TRIGGER = 'spinnaker.core.pipeline.trigger.pubsub';
+module(PUBSUB_TRIGGER, [
+  SERVICE_ACCOUNT_SERVICE,
+  PIPELINE_CONFIG_PROVIDER,
+]).config((pipelineConfigProvider: any) => {
+  pipelineConfigProvider.registerTrigger({
+    label: 'Pubsub',
```

Comment:
Is this the label that shows up on the trigger selection dropdown? I don't remember. But the label for any visible text should be consistent (either `Pubsub` or `Pub/Sub`). You use `Pub/Sub` in the template below.